code/operators/operators.cpp

```diff
@@ -61,31 +71,28 @@ int main() {
   // add in place
   f = athird;
   f += f += 1;
-  std::cout << "One third plus one and added to itself: " << f << '\n';
+  printAndCheck("One third plus one and added to itself", f, Fraction{8, 3});
   // again, try to refactor and implement operator+ in terms of operator+=
 
   std::cout << std::boolalpha;  // print bools as 'true' or 'false' from now on
 
   // compare two fractions for equality, all of these should be 'true'
```

Comment:
```suggestion
  // more equality comparisons
```
We can drop the fact that those should return true. And students must now implement `opeartor==` sooner.